### ISSUE TITLE
Kan et al. Multi-omics profiling of younger Asian breast cancers reveals distinctive molecular signatures. Nat Commun. 2018.

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/datahub#373]
Please keep this checklist for curation of a new study.

- [x] [create a issue on datahub](https://github.com/cBioPortal/datahub/issues/new) before curating a study (one issue per study) and copy this checklist to the issue tracker
- [ ] List information of the dataset/paper in the issue, e.g. pmid, paper link, suppl file link
- [ ] Document the curation process, e.g. how and by whom the data was transformed
- [ ] Follow [the data checklist](https://github.com/cBioPortal/datahub/issues/248)
- [ ] Create a pull request to datahub once the data is curated
- [ ] Push to triage portal
- [ ] Import into  msk and public portal database
- [ ] Update [cBioPortal news](https://github.com/cBioPortal/cbioportal/blob/master/docs/News.md)

**Comment by jjgao** [comment]
https://www.ncbi.nlm.nih.gov/pubmed/29713003

- [ ] Does study name and study ID follow our convention? e.g. Tumor_Type (Institue, Journal Year); brca_mskcc_2015
- [ ] is study meta data complete? e.g. pmid, group of PUBLIC
- [ ] were all samples profiled with WES/WGS? If not, is gene panel file curated?
- [ ] are oncotree codes of all samples curated; Cancer Type and Cancer Type Detailed needs to be added in addition to Oncotree Code
- [ ] clinical sample and patient data with meta files
- [ ] mutations data with meta files
- [ ] MAF is based on hg19
- [ ] MAF with 2 isoforms: uniprot and mskcc
- [ ] CNA data with meta files
- [ ] CNA segment data with meta files
- [ ] Expression data including z-scores with meta files
- [ ] Case-lists for all profiles.
- [ ] Passes Validation
- [ ] Manual checking (Niki or JJ): Triage Portal

**Comment by jjgao** [comment]
An author of the paper Samir Lal contacted us for depositing the data. They will work with us on formating the data.

**Comment by samirlal2** [comment]
Thank you, this is very helpful. This is my first upload to cBioportal. I am currently assembling the data and will get the data ready in the required format for upload.

**Comment by samirlal2** [comment]
@jjgao  I have deposited the expression data Log2(TPM+1) and TPM and MAF file to the PFizer-boxfolder. Please confirm if you have received the data?

**Comment by samirlal2** [comment]
@jjgao   I have created a pull request for data upload..

Here is the requested information. 
**List information of the dataset/paper in the issue, e.g. pmid, paper link, suppl file link**
Paper link: https://www.nature.com/articles/s41467-018-04129-4
pmid:  29713003

 **Document the curation process, e.g. how and by whom the data was transformed**
 Both tumor and blood DNA were enriched for exonic regions using the SureSelect XT regent kit (G9611B, Agilent) and SureSelect XT Human All Exon V5 kit (5190-6210, Agilent). Sequencing libraries were constructed for an Illumina HiSeq 2500 systems (Illumina) and sequenced in 100-bp paired-end mode of the TruSeq Rapid PE Cluster kit and TruSeq Rapid SBS kit (PE-402-4001, Illumina). Exome sequencing reads were mapped to the hg19 reference genome using Burrows-Wheeler Aligner (BWA-0.7.5a). PCR duplicates were removed by picard-1.93 (http://broadinstitute.github.io/picard). The mapped reads near putative indels were realigned and base quality was recalibrated using the GATK-2.4-7 suite. We aimed for 100× mean target coverage for tumors and 50× for paired blood samples. For RNA-Seq, sequencing libraries were prepared using TruSeq RNA Sample Preparation kit v2 (RS-122-2001 and RS-122-2002, Illumina). Sequencing of the RNA libraries was performed on an Illumina HiSeq2500 in 100-bp paired-end mode of the TruSeq Rapid PE Cluster kit and the TruSeq Rapid SBS kit (Samsung Genome Institute).

Gene expression (TPM) was predicted from RNA-Seq data of 178 SMC samples (168 tumors, 10 normals) using RSEM, hg19 as the genome reference and the same gene model library as used by Omicsoft ( Performed by Ying Ding). TPM was Log2 transformed by Samir Lal.
in the paired mutation calling mode. Parameters were selected based on the DREAM-3 setting as a false positive filter as recommended by Varscan2. All germline and somatic mutations were annotated using the Ensembl Variant Effect Predictor. Somatic mutations were detected from WES data on 186 SMC matched tumor/normal samples using Varscan2 v2.4.1. CNV segmentation was then called by ExomeCNV v1.4 based on coverage estimates and further annotated to derive copy number value for each gene.
